app/Main.hs

```diff
@@ -8,20 +9,33 @@ import Data.Monoid ((<>))
 
 import Options.Applicative
 
-import Configuration.Dotenv (loadFile)
-
+import Control.Monad (void)
 import Control.Monad.IO.Class(MonadIO(..))
 
+import System.FilePath ((</>))
+import System.Directory (getCurrentDirectory)
 import System.Process (system)
 import System.Exit (exitWith)
 
+import Configuration.Dotenv
+import Configuration.Dotenv.Types
+
 data Options = Options
-  { files    :: [String]
-  , overload :: Bool
-  , program  :: String
-  , args     :: [String]
+  { program           :: String -- ^ Program to run with the load env variables
+  , args              :: [String] -- ^ Args for the program that is going to be executed
+  , dotenvFile        :: String -- ^ Path for the .env file
```

Comment:
Never mind.

This PR's initial comment doesn't link to any issues or contain an explanation, so I was confused on its purpose at first. I just realised it might be referring to another package named `dotenv-safe` (looks like a Node package).

Could you not still have multiple environment files and one example file?